### PR TITLE
Add support for ndb

### LINKED ISSUE #2: 500 error from the datadog handler
TypeError: GlobalStat is not JSON serializable

changed db to ndb

from google.appengine.ext.ndb import stats as db_stats

and 'timestamp' is a date so:

data_stats = db_stats.GlobalStat.query().get().to_dict()
data_stats['timestamp'] = str(data_stats['timestamp'])
stats['datastore'] = data_stats
            
Thanks

Uri

### PR DESCRIPTION
**Pull request by LeoCavaille**
Fixes #2

### CODE REVIEW

**Comment by LeoCavaille** [comment]
Catch import error, if none of the modules are here.

File: datadog.py [hunk 1]
```diff
@@ -71,7 +75,8 @@ def get_request_stats(after=None):
             'project_name': app_identity.get_application_id()
         }
         if flavor == 'services' or flavor == 'all':
-            stats['datastore'] = db_stats.GlobalStat.all().get()
+            stats['datastore'] = db_stats.GlobalStat.all().get().to_dict()
```
Comment (de1ux) [inline datadog.py]: `to_dict` is a method available to ndb models, not db. Would need to use db.to_dict(db_stats.GlobalStat.all().get()) for db

Comment (EllisShen) [inline datadog.py]: Leo, can you add {{db.to_dict(db_stats.GlobalStat.all().get())}} for db into your branch?

**Comment by yannmh** [comment]
@LeoCavaille Is this PR still topical ? :sunny:

**Comment by LeoCavaille** [comment]
@yannmh I think so, it should also probably include the changes noted by the contributors.

**Comment by yannmh** [comment]
Thanks @LeoCavaille. I am closing in favor of https://github.com/DataDog/gae_datadog/pull/7

**Pull request closed without merging**